cmd/neofs-cli/internal/common/eacl.go

```diff
@@ -14,33 +14,34 @@ import (
 var errUnsupportedEACLFormat = fmt.Errorf("%w: unsupported eACL format", errors.ErrUnsupported)
 
 // ReadEACL reads extended ACL table from eaclPath.
-func ReadEACL(cmd *cobra.Command, eaclPath string) eacl.Table {
+func ReadEACL(cmd *cobra.Command, eaclPath string) (eacl.Table, error) {
 	_, err := os.Stat(eaclPath) // check if `eaclPath` is an existing file
 	if err != nil {
-		ExitOnErr(cmd, "", errors.New("incorrect path to file with EACL"))
+		return eacl.Table{}, WrapError("", errors.New("incorrect path to file with EACL"))
```

Comment:
here too enough to `return eacl.Table{}, errors.New("incorrect path to file with EACL")` and let caller how to wrap it. With this, u wont need to wrap all function returns